src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -6,33 +6,37 @@
  **/
 class InputWhitespaceAroundBasic
 {
-    /** warn **/
-    private int mVar1= 1; // warn
-    /** warn **/
-    private int mVar2 =1; // warn
+    private int mVar1= 1; // violation ''=' is not preceded with whitespace.'
+    private int mVar2 =1; // violation ''=' is not followed by whitespace.'
     /** Should be ok **/
     private int mVar3 = 1;
+    int xyz;       // multiple space between content and double slash.
+    int abc; //       multiple space between double slash and comment's text.
+    int pqr;       //     testing both.
```

Comment:
added test cases mentioned in https://github.com/checkstyle/checkstyle/issues/14932